native/ex_vl_convert/src/lib.rs

```diff
@@ -0,0 +1,261 @@
+use vl_convert_rs::converter::VgOpts;
+use vl_convert_rs::converter::VlOpts;
+use vl_convert_rs::VlConverter;
+use vl_convert_rs::VlVersion;
+
+// +-------------------------------------+
+// |        Rustler Return Types         |
+// +-------------------------------------+
+
+mod atoms {
+    rustler::atoms! {
+        ok,
+        error
+    }
+}
+
```

Comment:
I think we don't need this

```suggestion
```

Reply:
Good call. Forgot to remove that dead code from the previous clean up.